pytest_pootle/fixtures/import_export_fixtures.py

```diff
@@ -65,6 +65,9 @@ def _cleanup():
 @pytest.fixture
 def en_tutorial_ts(settings, english_tutorial, ts_directory):
     """Require the en/tutorial/tutorial.ts store."""
+
+    english_tutorial.project.localfiletype = "ts"
+    english_tutorial.project.save()
```

Comment:
you are probs right - but that is how its done currently - and im reluctant to change as tbh it needs a bit of a rewrite.

This change only allows ts in that Project - which should have been happening anyway. Im going to raise a related but that Store.create should check for localfiletype (soon to be filetypes)